buildings/gui/bulk_load_changes.py

```diff
@@ -854,20 +901,58 @@ def geometry_changed(self, qgsfId, geom):
             iface.messageBar().pushMessage("INFO",
                                            "You've edited the outline to less than 10sqm, are you sure this is correct?",
                                            level=QgsMessageBar.INFO, duration=3)
-        result = result.fetchall()[0][0]
-        if self.edit_dialog.geom == result:
-            if qgsfId in self.edit_dialog.geoms.keys():
-                del self.edit_dialog.geoms[qgsfId]
+        result = result.fetchall()
+        if len(result) == 0:
+            iface.messageBar().pushMessage("INFO",
+                                           "You've tried to split the same feature twice, you must save the changes to the db between splitting the same feature.",
+                                           level=QgsMessageBar.INFO, duration=5)
             self.disable_UI_functions()
+            self.edit_dialog.btn_edit_reset.setEnabled(1)
+            return
         else:
-            self.edit_dialog.geoms[qgsfId] = self.edit_dialog.geom
-            self.enable_UI_functions()
-            self.populate_edit_comboboxes()
-            self.select_comboboxes_value()
+            result = result[0][0]
+            if self.edit_dialog.geom == result:
+                if qgsfId in self.edit_dialog.geoms.keys():
+                    del self.edit_dialog.geoms[qgsfId]
+                self.disable_UI_functions()
+            else:
+                self.edit_dialog.geoms[qgsfId] = self.edit_dialog.geom
+                self.enable_UI_functions()
+                self.populate_edit_comboboxes()
+                self.select_comboboxes_value()
 
         self.edit_dialog.activateWindow()
         self.edit_dialog.btn_edit_save.setDefault(True)
 
+    @pyqtSlot(int)
+    def creator_feature_added(self, qgsfId):
+        if qgsfId not in self.edit_dialog.added_building_ids:
+            self.edit_dialog.added_building_ids.append(qgsfId)
+        # get new feature geom
+        request = QgsFeatureRequest().setFilterFid(qgsfId)
+        new_feature = next(self.editing_layer.getFeatures(request))
+        self.new_attrs = new_feature.attributes()
+        if not self.new_attrs[5]:
+            iface.messageBar().pushMessage("INFO",
```

Comment:
Would be better to have the error dialog pop up, like what we have when editing a geometry in `add outline` mode (L370)